sops/3scale-scaling.md

```diff
@@ -0,0 +1,65 @@
+# 3scale scaling on Openshift 3.11, Integreatly 1.4.x
+This guide states the known limits of 3scale capacity on Openshift along with scaling options for 3scale Components.
+
+**Note :** Pod anti-affinity rule is in place across a number of 3scale components. This may result in limited scaling options.
+
+## 3scale Scaling Components
```

Comment:
Maybe rename this to Component Scaling Options? The more generic we make the headings the easier it will be to copy for other products and provide a template